source/Controlled.tsx

```diff
@@ -23,7 +23,8 @@ interface Props {
   portalEl?: HTMLElement
   scrollableEl?: HTMLElement | Window
   transitionDuration?: number
-  wrapStyle?: CSSProperties | undefined
+  wrapElement?: ReactType
+  wrapStyle?: React.CSSProperties
```

Comment:
This can be destructured off of `React` at the top. Also, auto-sorting in your editor should put the capital-letters one first (`ReactType` goes after `ReactNode` above)

Reply:
> Also, auto-sorting in your editor should put the capital-letters one first (ReactType goes after ReactNode above)

Interesting... this is not enabled in Visual Studio Code by default. Are you using a plugin for that in your editor?

Reply:
Nope, it's the default way that your OS should be sorting. In VIM, it's `:sort`, but the `sort` CLI command in bash/zsh (whatever terminal) works the same way.